api/src/main/java/gov/cms/ab2d/api/controller/JobCompletedResponse.java

```diff
@@ -58,30 +65,40 @@ public void setError(List<Output> error) {
         this.error = error;
     }
 
+    @Getter
     public static final class Output {
-        private String type;
-        private String url;
+        private final String type;
+        private final String url;
+
+        private final List<ValueOutput> extension;
 
-        public Output(String type, String url) {
+        public Output(String type, String url, List<ValueOutput> valueOutputs) {
             this.type = type;
             this.url = url;
+            this.extension = valueOutputs;
         }
+    }
 
-        public String getType() {
-            return type;
-        }
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @Getter
+    public static final class ValueOutput {
```

Comment:
`ValueOutput` does not clearly communicate the purpose of this class. How about renaming to `FileMetadata` ?